onyo/lib/onyo.py

```diff
@@ -38,47 +38,104 @@ def __init__(self, path: Union[Path, str], init: bool = False) -> None:
 
     @property
     def assets(self) -> set[Path]:  # pyre-ignore[11]
+        """
+        A `set` containing the `Path`s to all assets of an Onyo repository.
```

Comment:
One bit of missing information: are these paths:
- absolute
- relative to `Repo.root`
- relative to `Repo.opdir`